src/data/ACTIONS/root/AST.ts

```diff
@@ -161,9 +161,11 @@ export const AST = ensureActions({
 		icon: 'https://xivapi.com/i/003000/003140.png',
 		cooldown: 90,
 		statusesApplied: [
-			'COLLECTIVE_UNCONSCIOUS_SHIELD',
+			'COLLECTIVE_UNCONSCIOUS_DIURNAL_MITIGATION',
```

Comment:
This looks the same as the data in the `5.1` layer - intentional?

Reply:
Actually, when I try to add fewer to the Root layer, it tells me that the `5.1` layer can't have anything but the original statuses applied.
```
TS2322: Type '{ statusesApplied: ("COLLECTIVE_UNCONSCIOUS_DIURNAL_MITIGATION" | "COLLECTIVE_UNCONSCIOUS" | "WHEEL_OF_FORTUNE_DIURNAL" | "COLLECTIVE_UNCONSCIOUS_NOCTURNAL" | "WHEEL_OF_FORTUNE_NOCTURNAL")[]; }' is not assignable to type 'Partial<{ id: number; name: string; icon: string; cooldown: number; statuldown: number; st
atusesApplied: ("COLLECTIVE_UNCONSCIOUS_DIURNAL_MITIGATION" | "COLLECTIVE_UNCONSCIOUS" | "WHEEL_OF_FORTUNE_DIURNAL")[]; } & Action>'.
```

Reply:
Also, the name change was to differentiate the Diurnal shield with the Nocturnal shield - mitigation is more apt since it's not actually a shield in both cases

Reply:
ergh, goddamnit. that's an issue in the typings. i'll have a look into it tonight.